<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>851</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="surnameEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Фамилия И.О.</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="createButton">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>710</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>750</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previewButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>790</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Предпросмотр</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileNameLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>780</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="hwidLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>730</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="therapistEdit">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Фамилия И.О.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="headOfDepartmentEdit">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>200</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Фамилия И.О.</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="complaintsEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Жалобы</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="anamnesisMorbiEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Anamnesis morbi</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="anamnesisVitaeEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Anamnesis vitae</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="stLocalisEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>144</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>St. localis</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="diagnosisEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>330</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Диагноз</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="examinationPlanEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>440</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>План обследования</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="treatmentPlanEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>580</y>
        <width>331</width>
        <height>111</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>План лечения</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>surnameEdit</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>previewButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
